popularity-calculation-script/src/popularity_scraper.py

```diff
@@ -1,111 +1,99 @@
 import json
 import re
-from urllib.request import urlopen
 
 import pandas as pd
+import requests
 
 
-class PopularityScraper:
-    def __init__(self, url: str, filename: str) -> None:
+class PopularityCalculator:
+    def __init__(self, url: str):
         self.url = url
-        self.filename = filename
         self.df = pd.DataFrame()
 
-    def download_data_from_kiwix_url(self, label="download.kiwix.org") -> dict:
-        """Get the relevent dictionery from the list of dictioneries from the data
-
-        Args:
-            label (str, optional): Label key of dictionary. Defaults to "download.kiwix.org".
-
-        Returns:
-            dict: the inner dictionary with the relevent label
-        """
-        response = urlopen(self.url)
-        data_json = json.loads(response.read())
+    def __download_data_from_url(self):
+        """Download data from url and append it to the Pandas DataFrame"""
+        response = requests.get(self.url)
+        data_json = json.loads(response.content)
         for i in data_json:
```

Comment:
poor choice of variable name. should describe what it represent. If not important, `row` is more descriptive for instance.

Reply:
Removed it

Reply:
Nope, it's still there: `for i in data_json:`